### PR TITLE
reload via page to prevent auth issues

### PR DESCRIPTION
**Pull request by fiji-flo**

### CODE REVIEW

File: src/components/profile/edit/EditPersonalInfo.vue [hunk 1]
```diff
@@ -405,7 +405,7 @@ import EditPictureModal from './EditPictureModal.vue';
 import Fetcher from '@/assets/js/fetcher';
 
 // TODO: Why do we need failoverOnError here?
```
Comment (LeoMcA) [inline src/components/profile/edit/EditPersonalInfo.vue]: This can probably be removed

File: src/pages/PageSearchResult.vue [hunk 2]
```diff
@@ -84,7 +84,10 @@ import SearchResultList from '@/components/search/SearchResultList.vue';
 import SearchToggle from '@/components/search/SearchToggle.vue';
 import Fetcher from '@/assets/js/fetcher';
 
-const fetcher = new Fetcher({ failoverOn: [302] });
+const fetcher = new Fetcher({
+  isError: (e) =>
+    e instanceof TypeError && e.message.startsWith('NetworkError'),
```
Comment (LeoMcA) [inline src/pages/PageSearchResult.vue]: Will search throw errors which aren't NetworkErrors? Will other fetches throw errors which aren't NetworkErrors? If they do, do we want them to reload on all fetch errors, but only search to reload on NetworkErrors?

File: public/reload.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width,initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Open+Sans:400,700|Zilla+Slab"
```
Comment (LeoMcA) [inline public/reload.html]: It's probably cached already, but do we really need open sans on a page which users will hopefully see for as little time as possible.

File: src/main.js [hunk 4]
```diff
@@ -33,7 +34,7 @@ function errorHandler(error) {
       networkError.message.startsWith('NetworkError')) ||
       failoverOn.includes(networkError.statusCode))
```
Comment (LeoMcA) [inline src/main.js]: Do we want apollo queries to only reload on 302s, when we've removed that restriction everywhere else?

Comment (fiji-flo) [reply src/main.js]: Not happy to write this: I'm to scared to change this right now.

File: src/pages/PageSearchResult.vue (hunk 2 above)
Comment (fiji-flo) [reply src/pages/PageSearchResult.vue]: Mostly keeping previous behavior to avoid breaking things for now.

File: public/reload.html (hunk 3 above)
Comment (fiji-flo) [reply public/reload.html]: :+1:

**Pull request merged**